leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.kt

```diff
@@ -236,6 +265,39 @@ class LocationPickerActivity : AppCompatActivity(),
         track(TrackEvents.ON_LOAD_LOCATION_PICKER)
     }
 
+    private fun setUpContentView() {
+        if (isLegacyLayoutEnabled) {
+            setContentView(R.layout.leku_activity_location_picker_legacy)
+        } else {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+                window.setFlags(WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS, WindowManager.LayoutParams.FLAG_LAYOUT_NO_LIMITS)
+            }
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                var flags: Int = window.decorView.systemUiVisibility
+                flags = flags and View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR.inv()
+                window.decorView.systemUiVisibility = flags
+            }
```

Comment:
Just for future iterations, I think this should be defined by the theme used on your application even can be customized in the manifest.

Reply:
I'm agree with you, though it's tied to the design and doing it in a different way it could look weird. I need to test it a bit. Need more time 😩